src/exodus/test/wallet_tests.cpp

```diff
@@ -51,9 +51,9 @@ class TestWallet : public exodus::Wallet
         exodus::Wallet::ClearSigmaMintChainState(id);
     }
 
-    void SetSigmaMintUsedStatus(const exodus::SigmaMintId& id, bool isUsed)
+    void SetTransactionId(const exodus::SigmaMintId& id, uint256 tx)
```

Comment:
We don't need this anymore since the method in the base class now public.

Reply:
nice catch